crates/matrix-sdk/src/utils.rs

```diff
@@ -237,6 +246,16 @@ pub fn formatted_body_from(
     }
 }
 
+/// Construct a `404 NOT_FOUND` error response from the server, with the given
+/// message.
+pub(crate) fn not_found_error(message: String) -> HttpError {
```

Comment:
With the other suggestion, we could remove this.

Reply:
Removed.